EducationPortal.DAL/Entities/Book.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace EducationPortal.DAL.Entities
+{
+    public class Book : Material
+    {
+        // Fixed name of property
+        public int PageCount { get; set; }
+
+        public List<string> Authors { get; set; }
+
+        public string Format { get; set; }
+        
+        public DateTime Published { get; set; }
+
+        public override string MaterialType { get; set; } = "Book";
```

Comment:
The MaterialType can be changed? (Maybe you should remove the setter)

Reply:
fixed